docs/dns/domains.rst

```diff
@@ -27,8 +27,8 @@ A JSON object representing a domain has the following structure::
                     "6006 13 2 f34b75...",
                     "6006 13 4 2fdcf8..."
                 ],
-                "flags": 257,
-                "keytype": "csk"
+                "flags": 257,  # deprecated
+                "keytype": "csk"  # deprecated
```

Comment:
Proper JSON comments are using (afaik exclusively) `/* ... */`

Reply:
This also affects a later commit

Reply:
JSON has no comments